data/data-pipeline/data_pipeline/etl/score/config/datasets.yml

```diff
@@ -3,7 +3,8 @@ datasets:
   - long_name: "FEMA National Risk Index"
     short_name: "nri"
     module_name: national_risk_index
-    description: "Dataset from FEMA that identifies communities most at risk to 18 natural hazards."
+    description_short: "Dataset from FEMA that identifies communities most at risk to 18 natural hazards."
+    description_long: "Dataset from FEMA that identifies communities most at risk to 18 natural hazards."
```

Comment:
This is not actually what I meant, sorry. 

There is a dataset description which is helpful. 

But for the short/long description fields, I was referring to the short/long description fields *per field*. So these would go into each of the `load_fields`. For instance, for the `load_field` that has `long_name: "Expected agricultural loss rate (Natural Hazards Risk Index)"`, this is the long description: https://screeningtool.geoplatform.gov/en/methodology#exp-agr-loss-rate:~:text=Percent%20of%20agricultural,that%20census%20tract. 

> Percent of agricultural value at risk from losses due to fourteen types of natural hazards that have some link to climate change: avalanche, coastal flooding, cold wave, drought, hail, heat wave, hurricane, ice storm, landslide, riverine flooding, strong wind, tornado, wildfire, and winter weather. Rate calculated by dividing the agricultural value at risk in a census tract by the total agricultural value in that census tract.

And this is the short description (from the sidebar): 

<img width="649" alt="Screen Shot 2022-08-02 at 5 17 34 PM" src="https://user-images.githubusercontent.com/25443529/182475037-d940f23e-abcf-4717-a79d-4dd254fb6e3c.png">

> Economic loss rate to agricultural value resulting from natural hazards each year.

Reply:
Got it! Will update.